spec/models/user_spec.rb

```diff
@@ -6,4 +6,19 @@
   it { should validate_presence_of(:full_name) }
   it { should validate_uniqueness_of(:email) }
   it { should have_many(:queue_items).order(:position) }
+
+  describe "#queued_video?" do
+    it "returns true when the user queued the video" do
+      user = Fabricate(:user)
+      video = Fabricate(:video)
+      Fabricate(:queue_item, user: user, video: video)
+      user.queued_video?(video).should be_true
```

Comment:
I think you should be using `expect(...).to` syntax rather than `.should`. It is a newer and preferred way RSpec goes now.